bin/vendor-react.js

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env node
+
+/* eslint-disable no-console */
+
+const {cp, set} = require("shelljs");
+const path = require("path");
+
+const filesToVendor = {
+  // XXX currently these two licenses are identical.  Perhaps we should check
+  // in case that changes at some point in the future.
+  "react/LICENSE": "REACT_AND_REACT_DOM_LICENSE",
+  "react/umd/react.production.min.js": "react.js",
+  "react/umd/react.development.js": "react-dev.js",
+  "react-dom/umd/react-dom.production.min.js": "react-dom.js",
+  "react-dom/umd/react-dom.development.js": "react-dom-dev.js",
+};
+
+set("-v"); // Echo all the copy commands so the user can see what's going on
+for (let srcPath of Object.keys(filesToVendor)) {
+  cp(path.join("node_modules", srcPath),
+    path.join("vendor", filesToVendor[srcPath]));
```

Comment:
Could probably have just shoved these into a single npm run vendor-react without a separate `bin/.js` but I guess this is more readable. 😉 

Still could be useful to expose a run script.

Reply:
What can I say, I'm a readability nerd. :-)  Good idea, though; I'll add it to
yamscripts.yaml for discoverability.